rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1934,6 +1934,12 @@ def test_upgrade_db_39_to_40(user_data_dir):  # pylint: disable=unused-argument
         accounting_treatment=None,
     ).serialize() == TxEventSettings.deserialize_from_db(accounting_row[4:]).serialize()
 
+    # check that the replacement for the VELO asset worked
+    assert cursor.execute(
```

Comment:
this test is incomplete. Would need at least 1 history event (or even more tables that have an FK relation) with `VELO`. And make sure that it gets replaced with the other asset

Reply:
diff 

```
INSERT INTO history_events(identifier,entry_type,event_identifier,sequence_index,timestamp,location,location_label,asset,amount,usd_value,notes,type,subtype) VALUES(10000,1,'RE_TEST_VELO',0,1658912400030,'A',NULL,'VELO','1','0','Test velo upgrade','deposit','spend');
INSERT INTO history_events(identifier,entry_type,event_identifier,sequence_index,timestamp,location,location_label,asset,amount,usd_value,notes,type,subtype) VALUES(10001,1,'RE_TEST_VELO_BNB',0,1658912400040,'A',NULL,'eip155:56/erc20:0xf486ad071f3bEE968384D2E39e2D8aF0fCf6fd46','1','0','Test velo upgrade','deposit','spend');
INSERT INTO trades(rowid,id,timestamp,location,base_asset,quote_asset,type,amount,rate,fee,fee_currency,link,notes) VALUES(1,'1a1ee5',1663553847,'A','VELO','EUR','A','0.0200000000','1300','0.0416','EUR','S7C2I61663553847','');
```